logic/scorer/scorer_test.go

```diff
@@ -10,6 +11,295 @@ import (
 	"github.com/joshprzybyszewski/cribbage/model"
 )
 
+func BenchmarkHandPoints(b *testing.B) {
+	hand := make([]model.Card, 0, 5)
+	seen := make(map[model.Card]struct{}, 5)
+	for len(hand) < 5 {
+		c := model.NewCardFromNumber(rand.Intn(52))
+		if _, ok := seen[c]; ok {
+			continue
+		}
+		hand = append(hand, c)
+		seen[c] = struct{}{}
+	}
```

Comment:
:cactus:  this would be a beneficial helper func, probably: `getRandomHand(numCards int) []model.Card`

Reply:
:nods: I also had the 💡 that we could add some entropy to our scoring tests by making `parseHand` (below) randomize the card order for each test run to make sure we aren't relying on setup order